chartjs-plugin-annotation.js

```diff
@@ -129,9 +129,11 @@ module.exports = function(Chart) {
 			}
 		},
 		destroy: function(chartInstance) {
-			var deregisterers = chartInstance.annotation.onDestroy;
-			while (deregisterers.length > 0) {
-				deregisterers.pop()();
+			if (chartInstance && chartInstance.annotation) {
```

Comment:
Do you actually need `chartInstance && chartInstance.annotation` or can you just check for `chartInstance.annotation`?